docs/section10/10-14.md

```diff
@@ -0,0 +1,177 @@
+                    - Check if the modeled efficiency meets the expected baseline efficiency: `efficiency_matches_acceptable_value = expected_baseline_eff_b == modeled_efficiency_b`  
+        - SYSTEM 4
+        - Else, system type is 4:`Else:` 
+            - Get dictionary list of baseline zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_b = get_hvac_zone_list_w_area (B_RMR)`   
+            - Get list of zones that the HVAC system serves (this is to get the zone associated with sys 3 or 4): `zone_list_b = hvac_zone_list_w_area_dict_b[hvac_b.id]["ZONE_LIST"]`  
+            Get the cooling capacity (needed for determining heating efficiency for system 4)
+            - Check if rated cooling capacity does not equal null, if it does not equal null then set the cooling capacity equal to this data element's value: `if cooling_system_b.rated_total_cool_capacity != Null:`
+                - Set cooling capacity equal to the rated cooling capacity: `total_cool_capacity_b = cooling_system_b.rated_total_cool_capacity`
+            - Else if, check that design cooling capacity does not equal null:`Elif cooling_system_b.design_total_cool_capacity != Null:`  
+                - Set cooling capacity equal to the design cooling capacity: `total_cool_capacity_b = cooling_system_b.design_total_cool_capacity`
+            - Else, set the was_capacity_not_defined boolean variable equal to TRUE: `Else: was_cool_capacity_not_defined = TRUE`  
+          
+            - SYSTEM 4 <65000  
+            - Check if cooling capacity is <65,000 Btuh OR the zone aggregation factor was defined and the capacity is less than 65000 Btuh OR the capacity was not defined (this is one of the most stringent) OR the aggregation factor associated with the zone equals null: `if (zone_list_b[0].aggregation_factor != Null AND total_heating_capacity_b/zone_list_b[0].aggregation_factor < 65000) OR total_cooling_capacity_b < 65000 OR was_cool_capacity_not_defined == TRUE OR zone_list_b[0].aggregation_factor == Null:`  
+                - Set the expected baseline efficiency for <65000 (if the modeled part OR full load efficiency values equals 3.4 then it passes; this is captured in the subsequent logic): `expected_baseline_eff_b = 3.4`  
+                    - Loop through each of the efficiency_metric_types associated with the heating system until PART_LOAD_EFFICIENCY_NO_FAN  is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b. : `For x in range(len(heating_system_b.efficiency_metric_types)):`
+                        - Check if the efficiency metric equals HEAT_PUMP_COEFFICIENT_OF_PERFORMANCE_HIGH_TEMPERATURE_NO_FAN : `if heating_system_b.efficiency_metric_types[x] == "HEAT_PUMP_COEFFICIENT_OF_PERFORMANCE_HIGH_TEMPERATURE_NO_FAN":` 
+                            - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`  
```

Comment:
Would like to discuss this. Should we assume the rating condition for the <65,000 Btuh category is 47F db? It is not clear since the resting standard is AHRI 210/240 which performances seasonal tests. I assume the intent of the 90.1 committee is that the rating condition be 47F but I do not see anything definitive.

Reply:
Yes, per group use 47F.